docs/valkyrie/odin/deployment/aws/worker.md

```diff
@@ -0,0 +1,23 @@
+The Odin worker is deployed in the AWS ec2 without a public endpoint. Workers are run in aws spot fleet, this fleet help us maintain minimum number of workers and are scalable as utilization.
+## Prerequisites
+- Container Engine (Podman/Docker) installed and running on your system
+- Golang installed
+
+## Worker setup: 
+- AWS spot fleet with a minimum worker count of 1 is required.
+  Fleet instances can be of any ec2 familiy
+
+- Instance groups are created with a preemptable VMs [(spot vm)](../../../../blog/posts/spot_instances.md)
```

Comment:
Remove references to blog posts, the blog posts will mostly be around why we chose a certain thing, they will not contain detailed setup steps